### ISSUE TITLE
MESFusion

### ISSUE THREAD

**Issue by ANGLE404** [cc99forever/CHTMAE99#1]
希望得到您的开源代码
1034874729@qq.com
您的论文是非常出色的成果

Hope to get your open source code
 1034874729@qq.com
 Your thesis is a very outstanding result

The model is trained on an NVIDIA GTX 3090 GPU on the PyTorch platform. During the training phase, the learning rate is set to 5 × 10−4, and the SGD optimizer is used. The batch size is set to 128, and the training runs for 20 epochs. The patch size for the channel Swin-Transformer is set to 7. Leave-one-out-of-subject cross-validation (LOSO) [[27](https://link.springer.com/article/10.1007/s40747-025-01855-3#ref-CR27)] serves as the validation method. The average length of MEs on the CAS(ME)2 dataset is 16, and the average length on the SAMM-LV dataset is 76. The distance between two frames in optical flow calculation is the half of the average length. The source code for this model is available on: https://github.com/cc99forever/MESFusion.
